SentryCapacitor.podspec

```diff
@@ -17,15 +17,19 @@ Pod::Spec.new do |s|
 
   if File.exist?('../../@capacitor/core/package.json') == false
     # If Capacitor was not found (could happen when using Yarn PNP), fallback to the
-    # required minimum version of Capacitor 4.
-    miniOSVersion = '13.0'
+    # required minimum version of Capacitor 7.
+    miniOSVersion = '14.0'
```

Comment:
This could be potentially breaking for some users, but since it's the fallback I think it's okay.